src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_format.py

```diff
@@ -56,7 +56,8 @@ def transform_blob_output(result):
         ('Blob Tier', 'properties.blobTier'),
         ('Length', 'properties.contentLength'),
         ('Content Type', 'properties.contentSettings.contentType'),
-        ('Last Modified', 'properties.lastModified')
+        ('Last Modified', 'properties.lastModified'),
+        ('Snapshot', 'snapshot')
```

Comment:
Does the stack version of storage support snapshot?  If not, this will likely break on stack.

Reply:
Yep, it does :)